bin/createNavData.js

```diff
@@ -11,7 +11,8 @@ function getPageTitle(content, path) {
   try {
     return content.match(metaTitleRE)[1].trim();
   } catch (err) {
-    throw `Error reading the frontmatter from "${path}/index.md"`;
+    throw `Error reading the frontmatter from "${path}/index.md"
+          due to exception ${err}`;
```

Comment:
Thanks for fixing this @RohanGurjar, I apologize for waiting so long to review. This looks great, apart from this line.

Generally speaking, showing the exact error is helpful, but it can be confusing in this case. The person seeing the error likely only wrote articles and didn't touch any code, so an error that `[1] is undefined` (something like that, I'm not sure what the exact error would be 😄) is more confusing and less helpful than just the first line ("error reading frontmatter").

Apart from that, I think this is good to go! Nice work 💯

Reply:
@systimotic I will revert the error message change and update the PR. Thank you :+1: